src/matcha_ml/state/matcha_state.py

```diff
@@ -7,9 +7,7 @@
 class MatchaStateService:
     """A matcha state service for handling to matcha.state file."""
 
-    matcha_state_dir = os.path.join(
-        ".matcha", "infrastructure", "resources", "matcha.state"
-    )
+    matcha_state_dir = os.path.join(".matcha", "infrastructure", "matcha.state")
```

Comment:
Will this change from PR 124?

Reply:
Depending on the order merged there will be a conflict, but yes both the name and the path will be updated.